README.md

```diff
@@ -1 +1,53 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+
+>This is  my personal Porto folio on which you can find my recent work and reach out to my social media platforms.
+
+### Built with 
+
+-HTML  
+-CSS  
+-Technologies used: LINT, HINT, LINTHOUSE 
+
+### Llive Demo 
+
+-N/A
+
+## Getting Started 
+
+-Clone the project via terminal or download the package.  
+-Open index.html file via browser  
+
+## Prerequisites
+
+- Install any supported Browser
+
+## Set up
+-npx stylelint "**/*.{css,scss}" npx hint ./index.html
+
+## Usage
+
+Open index.html located at the route folder of the project
+
+## Authors
+
+## MBONABUCYA James
+
+GitHub: @github.com/mbonabucya  
+Twitter: @Mbonabucya12  
+LinkedIn: https://linkedin.com/james-mbonabucya
```

Comment:
- [ ] Please, fix your contacts for something like this:
`GitHub: [mbonabucya](https://github.com/mbonabucya)`